src/assets/css/reset.css

```diff
@@ -0,0 +1,12 @@
+@import url('https://fonts.googleapis.com/css2?family=Space+Mono:wght@400;700&display=swap');
```

Comment:
We could put font import in global.js because it's not used in style reset. Or we can use this to set the font for `body` instead of `.container`.